runtimes/bundles/org.jboss.tools.rsp.server.wildfly/src/main/java/org/jboss/tools/rsp/server/wildfly/runtimes/download/DownloadManagerDownloadExecutor.java

```diff
@@ -225,7 +227,7 @@ private WorkflowResponse requestCredentials(String prefix, long requestId) {
 	}
```

Comment:
a l64 there's a System.out.println, shouldn't we replace this by proper logging?